Client/WarFare/UIImageTooltipDlg.cpp

```diff
@@ -1052,6 +1053,79 @@ int	CUIImageTooltipDlg::CalcTooltipStringNumAndWrite(__IconItemSkill* spItem, bo
 	}
 	ERROR_EXCEPTION
 
+	if (spItem->pItemBasic != nullptr)
+	{
+		std::string szRemark = spItem->pItemBasic->szRemark;
```

Comment:
We're not modifying this, so we don't really need to copy it. For aliasing, just use a reference, like:
```cpp
const std::string& szRemark = spItem->pItemBasic->szRemark;
```